### DISCUSSION TITLE
referencing a ResourceDictionary XAML file in a different library from a XAML file

### DISCUSSION THREAD

**Discussion by c-npolyak** [unoplatform/uno#11622] [General]
Friends, my XAML file is trying to reference a ResourceDictionary from a different library and it fails under Uno/WinUI
Both XAML files are essentially empty. 
Here is how I refer to the file in the library:

    <Page.Resources>
        <ResourceDictionary>
            <ResourceDictionary.MergedDictionaries>
                <ResourceDictionary Source="ms-appx:///MyLibrary/MyResources.xaml" />
            </ResourceDictionary.MergedDictionaries>
        </ResourceDictionary>
    </Page.Resources>

And here is the content of the file I trying to reference:

    <ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml%22%3E

    </ResourceDictionary>

Compilation runs fine, but when I try running the (essentially empty) application, I am getting Microsoft.UI.Xaml.Markup.XamlParseException: 'XAML parsing failed.'

**Comment by jeromelaban** [comment]
Thanks for the question. 

Given the information you provided, it should work properly with the WinAppSDK head. If you can provide a simple repro sample of what you're trying to do, it will help. Does it work with non windows targets?

**Reply by c-npolyak** [reply]
Let me try to create a small Github sample for you.

**Reply by c-npolyak** [reply]
My stand alone project that I wanted to show you, seems to be working. So apparently this is smth on my side. I am trying to figure it out.

**Reply by c-npolyak** [reply]
I made another project with the same functionality and it is working. Apparently somehow I inadvertently damaged the first project, so we can close this issue.
